### PR TITLE
Pica/Rasterizer: Add SecondaryFragmentColor

### PR DESCRIPTION
**Pull request by MoochMcGee**
Verified with the Dragon Quest 7 warning screen.

dq7_screen_1

### CODE REVIEW

File: src/video_core/rasterizer.cpp [hunk 1]
```diff
@@ -384,10 +384,13 @@ static void ProcessTriangleInternal(const VertexShader::OutputVertex& v0,
 
                 auto GetSource = [&](Source source) -> Math::Vec4<u8> {
                     switch (source) {
-                    // TODO: What's the difference between these two?
+                    // TODO: Implement fragment lighting. The following is a bunch of hacks.
```
Comment (bunnei) [inline src/video_core/rasterizer.cpp]: Not the best TODO, but meh, I think we all understand the issue here

**Comment by bunnei** [comment]
LGTM

File: src/video_core/rasterizer.cpp [hunk 2]
```diff
@@ -384,10 +384,13 @@ static void ProcessTriangleInternal(const VertexShader::OutputVertex& v0,
...
                     case Source::PrimaryColor:
                     case Source::PrimaryFragmentColor:
                         return primary_color;
+                    
+                    case Source::SecondaryFragmentColor:
+                        return 0;
```
Comment (Normmatt) [inline src/video_core/rasterizer.cpp]: shouldn't this return {} like in the default case?

Comment (neobrain) [inline src/video_core/rasterizer.cpp]: Indeed. This probably does the right thing, but tbh I'm not sure how it even compiles at all.

**Comment by bunnei** [comment]
@MoochMcGee please address the above comments

**Comment by neobrain** [comment]
@MoochMcGee Bump?

**Comment by MoochMcGee** [comment]
This PR is dead. Not sure if I have the code anymore.

**Comment by neobrain** [comment]
@MoochMcGee Please take more responsibility about this in the future instead of making us chase after you. Thanks for your contribution regardless.

**Pull request closed without merging**